### DISCUSSION TITLE
New version announced

### DISCUSSION THREAD

**Discussion by vascolp** [pybricks/support#545] [General]
I Pybrikers!

I noticed on FB that a new v3.1 release was announced... not yet announced here...
I just installed "v3.1.0c1 on 2021-11-19" on a TechnicHub, is it similar to the upcomming new version?
I tried to find a hub.name(new_name) method but it seems there is only a read name method.... is the hub set name method available on this beta version?
Also, you say "Improved motor and drivebase functionality". Is there a place with more details? I mean motors, I haven´t tried drivebase yet...

Thanks in advance!
VascoLP

**Comment by dlech** [comment]
> is the hub set name method available on this beta version?

You have to set the hub name before you flash the firmware. It can't be changed at runtime.

![image](https://user-images.githubusercontent.com/963645/143120274-4dae34be-7429-4e45-a0cb-ba6f4dc9db5d.png)

**Reply by vascolp** [reply]
Yes I noticed that one and used it immediatly :-) 
I thought I read you could change it from the script.... but it is powering off that you can.. my mistake sorry!

**Comment by laurensvalk** [comment]
> I just installed "v3.1.0c1 on 2021-11-19" on a TechnicHub, is it similar to the upcomming new version?

Yes, `v3.1.0c1` is like a preview of the new version. We want people to try this to see if there are any remaining small issues. Then we can still fix them for the upcoming official release.

> Is there a place with more details?

You can see a list of changes for every release by clicking on *changelog* in the *about* menu.:

![image](https://user-images.githubusercontent.com/12326241/143200568-9cef3de4-7d24-4640-9fa9-cea9d805932c.png)

> I mean motors

Here's a [list of motor fixes](https://github.com/pybricks/support/issues?q=is%3Aissue+is%3Aclosed+label%3A%22topic%3A+motors%22). You can find these in the changelog as well.

> Also, you say "Improved motor and drivebase functionality".

We've added the ability to drive in curve instead of only straight or turning in place. We still have to [document this](https://github.com/pybricks/support/issues/535).
